docs/installation/installation-using-docker-on-production-server.md

```diff
@@ -1,7 +1,10 @@
 # Installation Using Docker on Production Server
 
-This guide shows how to install and configure production server applications needed for running [shopsys/project-base](https://github.com/shopsys/project-base).
-We use docker containers for `php` applications and natively installed applications like `postgres`, `elasticsearch`, `redis` for persisting data without loosing them after `project-base` deployments.
+This guide shows you how to install and configure production server applications needed for your project based on [Shopsys Framework](https://github.com/shopsys/project-base).  
+We use natively installed applications like postgres, elasticsearch, redis for persisting data without loosing them after deploy of new version of the project.
```

Comment:
This sentence should be before the last one ("This guide also shows you how to setup first...")

Reply:
I would rewrite the sentence in this fashion: "As we do not want to lose data after deploying a new version of the project, we install all the data storages (postgres, elasticsearch, redis) natively though."